apps/omg_watcher/lib/omg_watcher/block_getter/supervisor.ex

```diff
@@ -37,7 +37,7 @@ defmodule OMG.Watcher.BlockGetter.Supervisor do
       }
     ]
 
-    opts = [strategy: :one_for_all, max_restarts: 1000, max_seconds: 60]
+    opts = [strategy: :one_for_all, max_restarts: 500, max_seconds: 5]
```

Comment:
I have lowered this because it was set to unrealistic numbers and would have never been propagated upwards. For example, if you don't set the `Infura` API key and let it crash with requests, it would never have propagated the error to the supervisor where an alarm is already raised for Ethereum connectivity.

Reply:
good call, this was annoying to say the least. Maybe we can drive it even lower, since proper mitigations of eth-node connectivity issues have been put in place?

Reply:
3 and 5? (the default)

Reply:
ok but just to confirm the "proper mitigations" part:

if, say, 6 of the `EthereumEventListeners` bork on `timeout` from geth or anything **at the same time** - the restarts will be held off till after geth is known to be responsive again and it won't kill the supervisor?

If we're not certain of this, then maybe a figure lower but still safely above the #`EthereumEventListeners` is a better idea.

OTOH if we such geth-related problems are going to be caught, then the default is probably the best, for simplicity sake.

Reply:
`if, say, 6 of the EthereumEventListeners bork on timeout from geth at the same time - the restarts will be held off till after geth is known to be responsive again and it won't kill the supervisor?` - Yes.

Reply:
perfect! then let's just do defaults here. (same for child chain server's supervisor or is it up-to-date already?)